sonic_service_client/dbus_client.go

```diff
@@ -203,10 +203,9 @@ func (c *DbusClient) HaltSystem() error {
     busPath := c.busPathPrefix + modName
     intName := c.intNamePrefix + modName + ".execute_reboot"
 
-    // Command to execute the system halt
-    command := "sudo reboot"
+    method := 3
```

Comment:
Can we have a documentation here for what this "3" means?

Reply:
Agree, request change here.